docs/java/api-reference.md

```diff
@@ -1,46 +0,0 @@
----
-id: api-reference-java
-title: Java API reference
-hide_title: false
-hide_table_of_contents: false
-sidebar_label: JavaDoc API Reference
-description: Check our generated API Documentation to discover SDK capabilities via code
-keywords:
-- sap
-- cloud
-- sdk
-- cloud native
-- cloud sdk
-- sap cloud sdk
-image:
----
-import MvnBadge from '../../src/sap/sdk-java/MvnBadge'
-
-The SAP Cloud SDK for Java is a toolkit to develop cloud native applications on the SAP Cloud Platform.
-
-## Should I update? ##
-We highly recommend regularly updating to the latest SDK version. It will help you:
```

Comment:
I find the note somewhat helpful, maybe we can move it somewhere else.

e.g. under a section in our introduction where we explain how often the SDK is released, what our schema is (SemVer) and what our policy on dependency updates is.

Reply:
I actually think I can make a heading section for release notes with useful info. I'll request another review when done.